lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +63,22 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+
+    result_dic = {}
+    for pair in word_frequencies.items():
+        word = pair[0]
+        count = pair[1]
+        for i in range(len(word) - 1):
+            token1 = word[i]
```

Comment:
what about slices?

Reply:
sorry i don't understand what slices do you mean...

Reply:
`pair = word[i:i+2]`